### PR TITLE
change by kirk

### PR DESCRIPTION
**Pull request by AndrewC243**
made another print statement

### CODE REVIEW

File: Hotel/src/main/java/org/example/Main.java
Comment (mario-gonzalez-h) [inline Hotel/src/main/java/org/example/Main.java:1]: Let him cook!

File: Hotel/src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package org.example;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("Hello world!");
+        System.out.println("Kir's change");
```
Comment (beatsnmelody) [inline Hotel/src/main/java/org/example/Main.java:6]: Hi kirk!

File: Hotel/.gitignore [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+### IntelliJ IDEA ###
+.idea/modules.xml
+.idea/jarRepositories.xml
+.idea/compiler.xml
+.idea/libraries/
+*.iws
+*.iml
+*.ipr
+
```
Comment (deasiadoss) [inline Hotel/.gitignore:14]: Awesome Code!

File: Hotel/.gitignore [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+### Eclipse ###
+.apt_generated
+.classpath
+.factorypath
```
Comment (mario-gonzalez-h) [inline Hotel/.gitignore:18]: Nice!

**Review by deasiadoss — APPROVED** [review]
:)

**Pull request merged**